### PR TITLE
add audio bitscope

### PR DESCRIPTION
**Pull request by privatezero**
Hopefully the explanation is coherent!

### CODE REVIEW

**Comment by ablwr** [comment]
👍  L 👍  G 👍  T 👍  M 👍

**Review by ablwr — APPROVED** [review]

**Comment by todrobbins** [comment]
👏🏻 Looks 👏🏻 Dope 👏🏻 To 👏🏻 Me👏🏻

**Review by todrobbins — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -631,6 +631,32 @@
...
 
+  <!-- abitscope -->
+  <span data-toggle="modal" data-target="#abitscope"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Audio Bitscope">Audio Bitscope</button></span>
+  <div id="abitscope" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Creates a visualization of the bits in an audio stream</h3>
+          <p><code>ffplay -f lavfi "amovie=<i>input_file</i>,asplit=2[out1][a],[a]abitscope=colors=purple|yellow[out0]"</code></p>
+          <p>This filter allows visual analysis of the information held in various bit depths of an audio stream. This can aid with identifying when a file that is nominally of a higher bit depth actually has been 'padded' with null information. The provided GIF shows a 16 bit WAV file (left) and then the results of converting that same WAV to 32 bit (right). Note that in the 32 bit version, there is still only information in the first 16 bits.
```
Comment (retokromer) [inline index.html]: closing `</p>` tag is missing

File: index.html [hunk 2]
```diff
@@ -631,6 +631,32 @@
...
+          <dl>
+            <dt>ffplay -f lavfi </dt><dd>starts the command and tells ffplay that you will be using the lavfi virtual device to create the input</dd>
```
Comment (retokromer) [inline index.html]: delete space before `</dt>`

**Review by retokromer — CHANGES_REQUESTED** [review]
Except those details LGTM. Thank you again!

**Review by retokromer — APPROVED** [review]

**Comment by retokromer** [comment]
Thank you!

**Pull request merged**